src/quo2/components/spacing.cljs

```diff
@@ -0,0 +1,16 @@
+(ns quo2.components.spacing)
+
+(def spacing
+  {:space-0  0
+   :space-1  2
+   :space-2  4
+   :space-3  6
+   :space-4  8
+   :space-5  10
+   :space-6  12
+   :space-7  16
+   :space-8  20
+   :space-9  24
+   :space-10 32
+   :space-11 40
+   :space-12 56})
```

Comment:
Instead of memorizing which `:spacing-d` to use, I think using integers as keywords is OK in this map. Also, I didn't find tokens for the spacing in Figma.
Something like
```clj
(def spacing
  {0  0
   2  2
   4  4
   6  6
   8  8
   10 10
   12 12
   16 16
   20 20
   24 24
   32 32
   40 40
   56 56})
```

Reply:
@ibrkhalil  We can find the spacing tokens here: https://www.figma.com/file/v98g9ZiaSHYUdKWrbFg9eM/Foundations?type=design&node-id=624-350&mode=design&t=xed0yo4wQPZIiuWa-0. Just to confirm, Do you mean to change for example: `:spacing-2` to just `:4` or something like `:spacing-4` instead?

Reply:
Hmm can you put these seconds in quo2.foundations.spacings to align with Figma? They should also keep the semantic name as to couple as described in figma foundations 👌

Reply:
cc @ilmotta as it's a topic we discussed recently to some degree

Reply:
I didn't see that spacing file before :) @rahulpsingh
Well as longs at it's tokenized then it's nice to have them tokenized here too!

Reply:
> Hmm can you put these seconds in quo2.foundations.spacings to align with Figma? They should also keep the semantic name as to couple as described in figma foundations 👌

Sure, yes!! I'll update the spacing to `quo2.foundations.spacings`.